files/en-us/web/accessibility/aria/aria_live_regions/index.html

```diff
@@ -246,8 +248,8 @@ <h3 id="Advanced_use_case_Roster"><span class="mw-headline" id="Use_Case:_Roster
 <p>Breakdown of ARIA live properties:</p>
 
 <ul>
- <li><code>aria-live="polite"</code> indicates that the screen reader should wait until the user is idle before presenting updates to the user. This is the most commonly used value, as interrupting the user with "assertive" might interrupt their flow.</li>
- <li><code>aria-atomic</code> is not set (<code>false</code> by default) so that only the added or removed users should be spoken and not the entire roster each time.</li>
+ <li><code>aria-live="polite"</code> indicates that the screen reader should wait until the user is idle before presenting updates to the user. This is the most commonly used value, as interrupting the user with <code>"assertive"</code> might interrupt their flow.</li>
```

Comment:
```suggestion
 <li><code>aria-live="polite"</code> indicates that the screen reader should wait until the user is idle before presenting updates to the user. This is the most commonly used value, as interrupting the user with <code>assertive</code> might interrupt their flow.</li>
```

Quotes are extraneous here.

Reply:
Same reasoning as an earlier comment. (I think keeping strings quoted helps newcomers understand the docs better.)